src/env-inl.h

```diff
@@ -40,6 +40,31 @@
 
 namespace node {
 
+namespace loader {
+inline PackageConfig::PackageConfig(PackageConfig&& pkg)
+    : exists(pkg.exists),
+      is_valid(pkg.is_valid),
+      has_main(pkg.has_main),
+      main(pkg.main),
+      type(pkg.type),
+      has_exports(pkg.has_exports),
+      exports((v8::Global<v8::Value>&&) pkg.exports) {
+}
+
+inline PackageConfig::PackageConfig(Exists exists, IsValid is_valid,
+                                    HasMain has_main, const std::string& main,
+                                    PackageType type, HasExports has_exports,
+                                    v8::Global<v8::Value>&& exports)
```

Comment:
```suggestion
                                    v8::Global<v8::Value>&& exports = v8::Global<v8::Value>())
```

I think this should let you get rid of manually using empty values below, if you like.

Reply:
Hm, now I'm tempted to also default `has_exports` to no exports so I can drop both arguments for the common case.

Reply:
Did that which means there's now fewer "random" edits in the other branches.